views/pages/partials/ga.ejs

```diff
@@ -0,0 +1,13 @@
+<!-- Google Analytics -->
+<script>
+  window.ga =
+    window.ga ||
+    function () {
+      (ga.q = ga.q || []).push(arguments);
+    };
+  ga.l = +new Date();
+  ga("create", "<%= gaId %>", "auto");
+  ga("send", "pageview");
+</script>
+<script async src="https://www.google-analytics.com/analytics.js"></script>
+<!-- End Google Analytics -->
```

Comment:
I'm not sure we need this as this will fire twice, as we can handle pageview using GTM only. see [this](https://www.youtube.com/watch?v=28d60ejfk3s) for more details

Reply:
We are not triggering through GTM